tests/mvp-demo/mvp-demo.go

```diff
@@ -334,50 +399,85 @@ var _ = framework.MvpDemoSuiteDescribe("MVP Demo tests", Label("mvp-demo"), func
 
 		})
 
-		It("JVM Build Service is used for rebuilding dependencies", func() {
+		It("application snapshot is created", func() {
+			snapshot, err = f.AsKubeAdmin.IntegrationController.GetApplicationSnapshot("", pipelineRun.Name, "", userNamespace)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(snapshot.Name).ToNot(BeEmpty())
+		})
+
+		It("Release is created and Release PipelineRun is triggered and Release status is updated", func() {
+			release, err = f.AsKubeAdmin.ReleaseController.GetRelease("", snapshot.Name, userNamespace)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(release.Name).ToNot(BeEmpty())
+
+			Eventually(func() bool {
+				pipelineRun, err = f.AsKubeAdmin.ReleaseController.GetPipelineRunInNamespace(managedNamespace, release.Name, release.Namespace)
+				if err != nil {
+					GinkgoWriter.Printf("pipelineRun for component '%s' in namespace '%s' not created yet: %+v\n", componentName, managedNamespace, err)
+					return false
+				}
+				return pipelineRun.HasStarted()
+			}, 2*time.Minute, 2*time.Second).Should(BeTrue())
```

Comment:
@kasemAlem don't we have constants for all these timeout values?

Reply:
I've added constants for timeouts and intervals for this test. Having constants for timeouts which would be used across all tests would be a good way to go, but it's probably something for a different PR ([example](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/timeouts.go))